docs/xtkJob.html

```diff
@@ -0,0 +1,131 @@
+---
+layout: page
+title: XTK Jobs (xkt:job, xkt:jobInterface)
+---
+<p>This section describes how long running jobs are handled in ACC and ACC api. There are 2 schemas for jobs</p>
+<ul>
+  <li>xtk:job which represents an acutal job, and which is implemented by several objects such as nms:delivery</li>
+  <li>xtk:jobInterface which contains methods to submit jobs and get their statuses. Other schemas do not actually implement the xtk:jobInterface interface</li>
+</ul>
+
+<p>In term of SOAP calls, running a job is simply executing a SOAP call synchronously or asynchronously in a separate thread or process. However, calls must be made on both the xtk:jobInterface and job entity schema. The SDK provides a helper interface to submit and handle jobs</p>
+
+<h1>Simple Jobs</h1>
+  
+<p>
+  Simple jobs are non-static methods with no parameters, such as the nms:delivery#Prepare method which is used to prepare a delivery. 
+  Such jobs can be executed synchronously (xtk:jobInterface#Execute) or asynchronously (xtk:jobInterface#Submit). Both methods return
+  a job id which can be used to retreive (poll) more about the job.
+<p>
+    
+<p>
+  First, we need to retreive a delivery, which is the object upon which to run the Prepare method, but also implements xtk:job interface. 
+  The Prepare method requires a complete delivery object, so we use SelectAll
```

Comment:
For my knowledge, does 'Prepare' method really require the complete delivery object? What happens if we only provide the keys?

Reply:
Yes, it does. The delivery object is passed directly to the JavaScript nms_delivery_Prepare(elDelivery) function.
Also, the job mechanism itself depends on the folder-id attribute and a couple of others.